lib/lualinux.c

```diff
@@ -113,6 +115,33 @@ static int lualinux_schedule(lua_State *L)
 	return 1;
 }
 
+/***
+* Kills a process by sending SIGKILL to pid.
+* @function kill
+* @tparam integer pid Process ID to kill.
+* @treturn boolean `true` if successful.
```

Comment:
if you raise an error, it doesn't make much sense to return true on success.. we should decide if it must raise an error or return false on failure.. what do you think?

Reply:
perhaps it's better to return true on success or false, err on failure.. what do you think?